CHANGELOG.md

```diff
@@ -1,5 +1,143 @@
 # Changelog
 
+## v1.11.0 (03.02.2021)
+
+### Enhancements
+
+#### :gear: Set Request Timeouts ([PR #5692](https://github.com/DevExpress/testcafe/pull/5692))
+
+TestCafe now enables you to set request timeouts. If a request is not met with a response within the specified period, TestCafe throws an error.
```

Comment:
What do you mean by "not met with a response"? Do you mean that the request does not receive a response? For example, can we say:

`If a request does not receive a response within the specified period, TestCafe throws an error.`

Reply:
Is it correct to say that a request receives a response? Or is it more correct to say `the issuer of the request receives a response`? In that case, can we say:

> If TestCafe receives no response within the specified period, it throws an error.

Reply:
> If TestCafe receives no response within the specified period, it throws an error.

I like your suggestion. Please apply it. Thanks.